src/functions-and-arrays.js

```diff
@@ -52,14 +163,47 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(unqArray) {
+  if (unqArray.length === 0) {
+    return null
+  }
+
+  if (unqArray.length >= 1) {
+
+    const newArray = []
+    unqArray.forEach(element => {
+
+
+      if (!newArray.includes(element)) {
+        newArray.push(element);
+      }
+
+    });
+    return newArray
+  }
```

Comment:
Bien resuelto. Nada más comenzar no se espera que encontréis una solución compleja a esta kata pero la forma más elegante (puedes investigarlo) de resolver este problema sería a través de un Set: https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Set

Un Set es una estructura de datos que alberga elementos ÚNICOS no repetidos.